runtime/gateway.go

```diff
@@ -62,20 +62,22 @@ type Gateway struct {
 	WaitGroup        *sync.WaitGroup
 	Channel          *tchannel.Channel
 	Logger           *zap.Logger
-	MetricScope      tally.Scope
+	RootScope        tally.Scope
```

Comment:
this is technically a breaking change.

Let's do `RootScope` , `MetricsScope` , `PerWorkerMetricsScope`